sentinel-adapter/sentinel-spring-webmvc-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/spring/webmvc/AbstractSentinelInterceptor.java

```diff
@@ -67,6 +67,7 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
             }
             return true;
         } catch (BlockException e) {
+            ContextUtil.exit();
```

Comment:
Maybe a `try-catch` here is better? Users may need the context in the block handler.

Reply:
Good idea, change to `try-finally`, which makes same effect as catch the Exeception of `handleBlockException` method and rethrow it, and it's suitable and concise here, please check.